### ISSUE TITLE
Login without eventpop

### ISSUE THREAD

**Issue by themarsbar** [dtinth/auden#21]
Hi, Is there a way to use this project without having eventpop integration?

**Comment by dtinth** [comment]
@themarsbar Yes, although you have to edit the code a little bit. It is based on Firebase Authentication, so as long as you can get a user to log in, they can use the app.
